config/initializers/breakers.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+require 'rx/configuration'
+
+# Read the redis config, create a connection and a namespace for breakers
+redis_config = Rails.application.config_for(:redis).freeze
+redis = Redis.new(redis_config['redis'])
+redis_namespace = Redis::Namespace.new('breakers', redis: redis)
+
+# The Rx::Configuration class contains the host and path for Rx requests.
+# Create a matcher proc that returns true for POST requests to that host and path.
+rx_path = URI.parse(Rx::Configuration.instance.base_path).path
+rx_host = URI.parse(Rx::Configuration.instance.base_path).host
+rx_matcher = proc do |request_env|
+  request_env.method == :post &&
+    request_env.url.host == rx_host &&
+    request_env.url.path =~ /^#{rx_path}/
+end
```

Comment:
You know, I'm not really sure what I was thinking about only using it for POST... good catch.

I'm working on the Secure Messaging thing as a separate PR right now, and part of that will involves creating the services in the Configuration classes. It'll be a lot cleaner.